include/rocksdb/table_properties.h

```diff
@@ -232,6 +232,10 @@ struct TableProperties {
   // compression algorithm (see `ColumnFamilyOptions::sample_for_compression`).
   // 0 means unknown.
   uint64_t fast_compression_estimated_data_size = 0;
+  // Offset of the value of the property "external sst file global seqno" in the
+  // file if the property exists.
+  // 0 means not exists.
```

Comment:
Note to reviewer:
I think it is save to use 0 as the sentinel value for "not exists" since BlockBasedTable [writes](https://github.com/facebook/rocksdb/blob/main/table/block_based/block_based_table_builder.cc#L1711-L1716) a whole bunch of properties like `TablePropertiesNames::kDataSize` before writing `ExternalSstFilePropertyNames::kGlobalSeqno`, even in the empty sst file case (when data size = 0). Therefore by the `BytewiseComparator` here we know `ExternalSstFilePropertyNames::kGlobalSeqno` will be listed after `TablePropertiesNames::kDataSize` since [`rocksdb.data.size`](https://github.com/facebook/rocksdb/blob/main/table/table_properties.cc#L223-L224) < [`rocksdb.external_sst_file.global_seqno`](https://github.com/facebook/rocksdb/blob/main/table/sst_file_writer.cc#L24). Therefore we know if `ExternalSstFilePropertyNames::kGlobalSeqno` exists, this will be greater than 0.

Reply:
Works for me.